index.js

```diff
@@ -9,7 +9,7 @@ copyright.innerHTML = `Nequil &copy; ${thisYear}`;
 footer.appendChild(copyright); 
 
 const skills = ["JavaScript", "HTML", "CSS", "Git","GitHub"]; 
-const skillsSection = document.getElementById('Skills'); 
+const skillsSection = document.getElementById('skills'); 
```

Comment:
Your HTML is using Skills with a capital S. This is causing skillsSection to be null.